### PR TITLE
Fix: Remnant: Correcting principle to principal

### PR DESCRIPTION
**Pull request by Zitchas**
**Bugfix:**

## Fix Details
Lines 771 and 778 principle to principal.

Also rewording the sentence to be slightly shorter and flow a bit better.

Thank you to KarikVot on the Endless Sky Discord for spotting this!

### CODE REVIEW

File: data/remnant/remnant 2 cognizance.txt [hunk 1]
```diff
@@ -768,14 +768,14 @@ mission "Remnant: Cognizance 21"
...
+			`	In the cafeteria the scene hasn't changed much: There is still a large group of Remnant gathered around a cluster of projectors, with a constant stream of gestures discussing various points of the data. You understand enough to figure out that most of the conversation is deep into technical discussions of how the lifesign sensors might give the readings they do without having anything visible.The principal difference from before is the presence of five pressure-suited Remnant with a neat stack of equipment topped with helmets. One of them notices you and waves you over. "Captain <first>! My name is Dusk, and I am one of the researchers that you volunteered to transport to Ssil Vida."`
```
Comment (MCOfficer) [inline data/remnant/remnant 2 cognizance.txt:771]: Lost the space before "The principal"
```suggestion
			`	In the cafeteria the scene hasn't changed much: There is still a large group of Remnant gathered around a cluster of projectors, with a constant stream of gestures discussing various points of the data. You understand enough to figure out that most of the conversation is deep into technical discussions of how the lifesign sensors might give the readings they do without having anything visible. The principal difference from before is the presence of five pressure-suited Remnant with a neat stack of equipment topped with helmets. One of them notices you and waves you over. "Captain <first>! My name is Dusk, and I am one of the researchers that you volunteered to transport to Ssil Vida."`
```

**Review by MCOfficer — CHANGES_REQUESTED** [review]

**Review by MCOfficer — COMMENTED** [review]
One more thought, what's your take on "topped with" vs "topped by"? The former seems much more obscure to me.

**Comment by Zitchas** [comment]
"topped with" vs "topped by"? Rereading the sentence with each of those, they both feel about the same to me.

**Review by MCOfficer — APPROVED** [review]
alright, i'll let wizrad do the honors

**Review by W1zrad — APPROVED** [review]

**Pull request merged**